### PR TITLE
Fixed DNS popover alignment and link color

### PR DESCRIPTION
**Pull request by grolu**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix user

```

### CODE REVIEW

File: frontend/src/components/ShootDns/GDnsProvider.vue [hunk 1]
```diff
@@ -28,17 +28,19 @@ SPDX-License-Identifier: Apache-2.0
         />
       </v-chip>
     </template>
-    <v-list class="pa-0">
+    <v-list>
```
Comment (petersutter) [inline frontend/src/components/ShootDns/GDnsProvider.vue:31]: we should increase the min-width of the popover as the title could get cut off
Screenshot 2023-09-14 at 15 20 01

Comment (grolu) [reply frontend/src/components/ShootDns/GDnsProvider.vue]: done

**Review by petersutter — APPROVED** [review]
/lgtm

File: frontend/src/components/ShootDns/GDnsProvider.vue [hunk 2]
```diff
@@ -28,17 +28,19 @@ SPDX-License-Identifier: Apache-2.0
         />
       </v-chip>
     </template>
-    <v-list class="pa-0">
+    <v-list min-width="300px">
```
Comment (holgerkoser) [inline frontend/src/components/ShootDns/GDnsProvider.vue:31]: We usually do not use units for component widths.
```suggestion
    <v-list min-width="300">
```

**Pull request merged**